docs/lifecycle-events.md

```diff
@@ -25,7 +25,9 @@ Fired before the element is removed from the DOM.
 Note that when using this event you are responsible for removing the element yourself.
 
 ```js
-element.parent.removeChild(element)
+if (element.parentNode) {
```

Comment:
@SkaterDad BTW, I wasn't sure how to document this feature, but the second argument to the onremove function is a _function_ that can remove the element, so we can actually do:

```js
onremove(el, removeElement) => removeElement()
```